pumpkin-lib/src/api/solver.rs

```diff
@@ -649,7 +651,9 @@ impl Solver {
     }
 
     /// Post a new propagator to the solver. If unsatisfiability can be immediately determined
-    /// through propagation, this will return a [`ConstraintOperationError`].
+    /// through propagation, this will return a [`ConstraintOperationError`]. A tag can be
```

Comment:
I think we should split this into 2 methods since now we need to pass None to every call to this method in the normal API